proto/attest/attest.proto

```diff
@@ -160,8 +160,14 @@ message AttestStorageKeysResp {
   AttestStorageKeysResponseCode response = 1;    //Result of processing Z_ATTEST_REQ_TYPE_STORE_KEYS in Controller
 }
 
+message AttestPolicyPcrList {
+  repeated uint32 pcr_indices = 1; // list of PCR indices
```

Comment:
By default the protobuf decoder will decode a message from an old sender as having a default value for tag 1 and tag 2 (which is an empty pcr_indicies list and a zero policy_id). 

How can you tell such an old/empty/default message from a message which wants to include no pcr_indicies and with a zero policy_id?

One possibility would be to require that the policy_id always be non-zero, but if that's the approach it needs to be documented here in the API.